core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -357,6 +357,9 @@ sealed private[data] trait WriterTFlatMap1[F[_], L] extends WriterTApply[F, L] w
   implicit override def F0: FlatMap[F]
   implicit def L0: Monoid[L]
 
+  override def ap[A, B](f: WriterT[F, L, A => B])(fa: WriterT[F, L, A]): WriterT[F, L, B] =
```

Comment:
Just curious, was this an optimization opportunity you happen to come across or was it broken on RC1?
Also would it be simpler to just call `fa.ap(f)`

Reply:
This is from @SethTisue's commit, but the issue is that 2.13 sees the `ap` definitions in `FlatMap` and `WriterTApply` as conflicting. I personally think picking out the appropriate implementation via `super` is the right thing to do in this case, but I also don't think it matters much, and am happy to change it to `fa.ap(f)` if you'd prefer.